src/styles/_catalog.scss

```diff
@@ -1,22 +1,34 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
-  margin: 0 auto;
+  width: 100%;
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
```

Comment:
If u use default value for amount of columns, it should be min: repeat(1, 200px) and then increase it to bigger screens with @media .